### PR TITLE
Image

### PR DESCRIPTION
**Pull request by parulgupta2001**

### CODE REVIEW

File: styles/component-style/image.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.round-img {
+    border-radius:50%;
+    border: solid 2px var(--color-two);
+    height:30rem;
+    width:30rem;
+    object-fit: cover;
+}
+
+
+
```
Comment (humzahasan) [inline styles/component-style/image.css]: Extra spaces can be removed.

File: styles/component-style/image.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.responsive-img-container{
+    width:100%;
+}
+
+.responsive-img {
+   width:70%;
```
Comment (humzahasan) [inline styles/component-style/image.css]: width with 100% will give better result.

Comment (rutu14) [reply styles/component-style/image.css]: +1  Give width 100% to the image instead of the container.

**Review by rutu14 — APPROVED** [review]

**Review by humzahasan — APPROVED** [review]
Good to Merge!

**Pull request merged**